pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.fr-fr.md

```diff
@@ -40,10 +64,28 @@ $ kubectl get nodes -o jsonpath='{ $.items[*].status.addresses[?(@.type=="Intern
 51.77.204.175 51.77.205.79
 ```
 
+#### Your cluster is installed with Private Network with a OVHcloud Internet Gateway (or a defaultVrackGateway if you are using OVHcloud APIv6)
+
+Because your nodes are configured to be routed by the private network gateway, you need to add the gateway IP address to the ACLs.
+By using Public Cloud Gateway through our Managed Kubernetes Service, Public IPs on nodes are only for management purposes: [MKS Known Limits](/pages/public_cloud/containers_orchestration/managed_kubernetes/known-limits)
+
+You can get your OVHcloud Internet Gateway's Public IP by navigating through the OVHcloud Manager:
```

Comment:
We should add here a more common method to guess the egress IP, like starting a pod and perform a 
```bash
curl ifconfig.me
```
We can keep the method which is using the OVHcloud Manager.